spec/std/crystal/compiler_rt/divmod128_spec.cr

```diff
@@ -0,0 +1,89 @@
+require "spec"
+
+# TODO: Remove these helpers in PR part 2
```

Comment:
I'm not sure if this TODO is really necessary. Using the helpers isn't that bad. It stays close to the original code.
If we keep it, it should reference after the next release. Nobody's gonna know what `PR part 2` means.

Reply:
Once a crystal release supports U/Int128 across all platforms, I think this should be changed to literals.
IMO "-170141183460469231731687303715884105728" is easier to understand than make_ti(-9223372036854775808, 0x0)

Reply:
Fixed in #11206